csm/core/agent/api.py

```diff
@@ -145,6 +145,21 @@ def http_request_to_log_string(request):
         user_agent = request.headers.get('User-Agent')
         return (f"Remote_IP:{remote_ip} Url:{url} Method:{method} User-Agent:{user_agent}")
 
+    @staticmethod
+    def process_audit_log(resp, request):
+        url = request.path
+        if (not request.app["usl_polling_log"]) \
+                and ("/usl/" in url) \
+                and (url not in ["/usl/v1/registerDevice"]):
```

Comment:
Please consider [these changes](https://github.com/Seagate/cortx-manager/pull/183#discussion_r501694542).

Reply:
I will change it to != as we have only one value which we want to allow

Reply:
Yes, but please consider the other changes as well. Explicit line breaks are not required as the whole expression is nested in a pair of parentheses. Besides that, `"/usl/" in url` could map to API calls outside the current USL namespace, if they are ever added.

Reply:
Sorry if I didn't make that clear at first, but it would be nice if we got rid of the redundant parentheses in the condition as well.

Reply:
Added explicit parenthesis and removed line break.  "usl" in url. don;t have any better solution.

Reply:
> Added explicit parenthesis and removed line break. "usl" in url. don;t have any better solution.

What about the approach in my example, `url.startswith("/usl/")`? Wouldn't that work?

Reply:
Yes certainly that's better missed that. Let the parenthesis be there.

Reply:
Updated with startswith